include/zephyr/bluetooth/conn.h

```diff
@@ -530,6 +579,112 @@ int bt_conn_get_remote_info(struct bt_conn *conn,
+ */
+int bt_conn_le_get_remote_tx_power_level(struct bt_conn *conn,
+					 enum bt_conn_le_tx_power_phy phy);
+
+/** @brief Enable transmit power reporting.
+ *
+ *  @param conn           Connection object.
+ *  @param local_enable   Enable/disable reporting for local.
+ *  @param remote_enable  Enable/disable reporting for remote.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ *  @retval -ENOBUFS HCI command buffer is not available.
+ */
+int bt_conn_le_set_tx_power_report_enable(struct bt_conn *conn,
+					  bool local_enable,
+					  bool remote_enable);
+
+/** @brief Set remote (peer) transmit power.
+ *
+ *  @param conn			Connection object.
+ *  @param phy			PHY bit number i.e. [1M, 2M, s8, s2] == [1, 2, 3, 4]
+ *  @param delta        Delta, as in the transmit power level adjustment request in dBm,
+ *                      to be used by remote to apply to its transmit power.
+ *                      Note that Delta is only a reqest to the peer, who is in control
+ *                      how, or if at all, to apply changes to his transmit power.
+ *                      Furthermore, it can be 0 and thus utilize the response of the peer
+ *                      to update information of the tx power setting of remote.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ *  @retval -ENOBUFS HCI command buffer is not available.
+ */
+int bt_conn_set_remote_tx_power_level(struct bt_conn *conn,
+					 enum bt_conn_le_tx_power_phy phy, int8_t delta);
+
+/** @brief Set Power Control request parameter
+ *
+ *  @param auto_enable       Enable or Disable controller initiated autonomous
+ *                           LE Power Control Request procedure.
+ *                           Disabled by default.
+ *  @param apr_enable        Enable or Disable APR, i.e. Acceptable Power Reduction, handling
+ *                           in controller during LE Power Control Request procedure.
+ *                           Disabled by default.
+ *  @param beta              Beta value used to determine weight of previous average of RSSI values.
+ *                           The higher beta, the lower the current RSSI weighs into the average,
+ *                           flattening peaks, which also means the controller reacts slower
+ *                           on RSSI changes as well.
+ *                           The valid range of beta is [0, 4095].
+ *                           In the controller the beta value is used in an exponential weighted
+ *                           averaging of RSSI in 12-bit fixed point fraction.
+ *                           avg[n] = gamma * avg[n - 1] + (1 - gamma) * rssi[n]
+ *                           With gamma being beta/4096, and rssi[n] the current RSSI.
+ *                           For example, for gamma to be 0.25,
+ *                           set the beta parameter in the command to 1024.
+ *                           Default value is 2048.
```

Comment:
```suggestion
 *  @param beta              Value used to determine the weight of the previous average of RSSI values.
 *                           A higher value lowers how much the current RSSI weighs into the average,
 *                           flattening peaks, which also means the controller reacts slower
 *                           on RSSI changes.
 *                           The valid range is [0, 4095].
 *                           Default value is 2048.
 *                           In the controller, the value is used in an exponential weighted
 *                           averaging of RSSI in a 12-bit fixed point fraction.
 *                           avg[n] = gamma * avg[n - 1] + (1 - gamma) * rssi[n],
 *                           where gamma equals beta/4096, and rssi[n] equals the current RSSI.
 *                           For example, for gamma to be 0.25,
 *                           set the beta parameter in the command to 1024.
```
Above is a simple edit.
However, this is an extremely long parameter definition. At the cost of lessening the readability of the file itself but increasing the readability of generated documentation, I would suggest using a `@parblock`:
```suggestion
 *  @param beta
 *  @parblock
 *  Value used to determine the weight of the previous average of RSSI values.
 *  A higher value lowers how much the current RSSI weighs into the average,
 *  flattening peaks, which also means the controller reacts slower
 *  on RSSI changes.
 *  The valid range is [0, 4095].
 *  Default value is 2048.
 *
 *  In the controller, the value is used in an exponential weighted
 *  averaging of RSSI in a 12-bit fixed point fraction.
 *
 *  avg[n] = gamma * avg[n - 1] + (1 - gamma) * rssi[n]
 *
 *  Here, gamma equals beta/4096, and rssi[n] equals the current RSSI.
 *  For example, for gamma to be 0.25,
 *  set the beta parameter in the command to 1024.
 *  @endparblock
```

Reply:
I went with the parameter block.